### DISCUSSION TITLE
[Feature] iOS Upload only when WiFi Network is connected

### DISCUSSION THREAD

**Discussion by Happyfeet01** [immich-app/immich#6316] [Feature Request]
### The feature

Hello, 

is it possible to add the option, upload only when connected to an WiFi Network, the upload starts? The request is for iOS Devices. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
This can already be done by setting your phone's Background App Refresh setting in the Settings.

**Reply by Happyfeet01** [reply]
The background settings are enabled.

**Comment by aisbergde** [comment]
I think, this request is to avoid not only background upload activities, but all upload activities, when not on wifi. Or to stop them, when started with wifi, and wifi is lost.

**Reply by claude-gagne** [reply]
I would rather not spend all my cellphone data to upload videos and pictures. It would be nice that it only uploads when we are connected to a Wifi.

**Comment by claude-gagne** [comment]
I think it's a duplicate of https://github.com/immich-app/immich/discussions/5096
